about.md

```diff
@@ -0,0 +1,217 @@
+
+### Markdown Makes Formatting Easy
+
+Part of Pico's simplicity comes from the use of Markdown for formatting your pages.  According to its creator, John Gruber, "Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML)."
+
+Writing in Markdown is a lot quicker than coding in HTML, and much easier to learn.  Markdown makes building new pages as easy as writing basic text.  A Markdown file is just a plain text document with an `.md` extension instead of `.txt`.  Even this page was written using the simplicity of Markdown!
+
+You can learn more about Markdown on [Wikipedia][WikiMarkdown].  There's also a [good example][WikiMarkdownExample] there showing the before and after of how Markdown works, so be sure to check that out!
+
+You can learn how to use Markdown in its official documentation at [Daring Fireball][Markdown].  Pico also uses the extensions to Markdown found in [Markdown Extra][MarkdownExtra].  Markdown is a very common formatting syntax, and you'll find there are many other pieces of software that use it as well.  The basic Markdown syntax is universal across almost all software, but many will add their own unique extensions (like Markdown Extra).  Just keep the differences in mind if you decide to learn these extra features, as just about everyone has their own "flavor" of Markdown.
+
+```
+## Markdown Heading
+
+This is an example of using Markdown to format your document.  It's not very difficult.
+
+* Here are some bullet points.
+* Markdown is pretty simple to understand...
+  * But it's also really readable.
+  * HTML can't even compare!
+* You'll get the hang of it in no time at all.
+```
+
+### Control Your Metadata With YAML
+
+As we covered earlier, all of Pico's content is stored in flat files.  This includes the Metadata for your pages as well.
+
+Metadata makes up all the little details that accompany your page.  Things like the title, author, description, post date, etc.  While other CMS's would store this data in the database with everything else, Pico simply places it at the beginning of the file.
+
+Each Markdown-based page you create in Pico will have a small header at the top that contains all the metadata for that file.  The metadata is formatted using [YAML][YAML], which makes it really easy to manage.  We go more in-depth about [using YAML][DocsYAML] for your page header in our Documentation.
+
+Let's say for instance that you have a page that you'd like to back-date to last January.  Maybe that was the date you started your project or organization and you want the page to reflect that.  All you do is... type in that date!  There's no hoops to jump through, you don't have to go into some management console to modify your post, you just change the date.  You also don't have to worry that that minor edit you made to an older file will silently change a page's date to today's, thoroughly confusing all your visitors.
+
+```
```

Comment:
Hmm... I think I disagree, mostly because Pico doesn't use any arrays in its Metadata by default.  Though this idea and the multi-line text would make good examples for the proper YAML section.

But, if I use "actual content" for the markdown example above, for consistency I should do so here as well.